### DISCUSSION TITLE
non_nullable_equals_parameter

### DISCUSSION THREAD

**Discussion by ferrarafer** [rrousselGit/freezed#1050] [Q&A] [answered]
Hi, I'm getting the following warning on all the generated **.freezed** files. Is there a way to remove the warning?

```
The parameter type of '==' operators should be non-nullable.
Try using a non-nullable type.
```

The warning is for the following line of code.

`bool operator ==(dynamic other) {`

**Comment by mono0926** [comment, marked as answer]
@ferrarafer 

In version 2.4.6, the generated code has been changed from `dynamic` to `Object`, resolving the issue.

**Reply by ferrarafer** [reply]
Thanks ❤️ Sorry for not seen the update
